pages/credits.md

```diff
@@ -1,12 +1,18 @@
+{% include nav.html %} {% include css.html %}
+
 # Credits
 
 ---
 
 ## Project Credits
 
-* Put your group member's credits here
-* Link to any emails or github accounts (if you want)
-* Leave the credits at the bottom
+* [Stephen Truman](mailto:s.tru989@gmail.com)
+    * In this project, I worked on the introduction and reflection. Additionally I also gathered the data used in the visulizations. The introduction focus on explaining the projected course of scholarship and provided some background information related to piracy and slavery. To achieve this, I worked closely with my group members to ensure that the introduction conformed to the study and findings from the historiography and data results provide by my teammates. Furthermore, this introduction explained the digital tools used for this project, providing brief introductions into the utilization of the tools used. Secondly, I was responsible for gathering data to provide our visualization specialist. The data I gathered focus primarily on interaction of piracy on the slave trade from the 1700-1750s to answer a specific question about how piracy effected the economy surrounding the west indies. 
+* [Joseph Perez](mailto:joemperez85@gamil.com)
+    * By gathering the data generated from the 8-10 sources, we will demonstrate fact-finding visualization comparing the relationship between pirating and slavery in the 1700’s West Indies. The economic relationship regarding the effects of these practices will be examined. There will be a written initial findings and problems encounters report along with diagrams generated using Tableau. Another program that will be used is Orange3 to show a precise relationship and some digital effects for the project. Once the data is scraped and forwarded to vitalization, the imagery is constructed and explained to the viewers. During the scraping and data generating process, the visual producer will review the previous class material along with tutorials to better understand how to input the visualization on Github in a manner that will make sense to the viewers.
```

Comment:
You might want to update these to past tense, sense you have completed the project